src/Pages/CropCycleInfoPage.js

```diff
@@ -262,17 +267,45 @@ function CropCycleInfoPage() {
    //Opens the edit modal and passes the relevant row information
    * @param {object} selectedRow 
    */
-  const editField = (selectedRow) => {
+  const editCropCycle = (selectedRow) => {
     setIsEditModalOpen(true);
     setSelectedRow(selectedRow);
   };
 
+  /**
+   * delete handler for crop cycle
+   * @param {object} selectedRow
+   */
+  const deleteCropCycle = (selectedRow) => {
+    setSelectedRow(selectedRow);
+    setIsDeleteModalOpen(true);
+  };
+
+  /**
+   * delete crop cycle after user confirmation
+   * @param {object} selectedRow
+   */
+  const deleteCropCycleConfirmed = () => {
```

Comment:
`deleteCropCycleConfirmed` doesn't seem good for a method name; it's more like a boolean name. I would suggest the following.
1. This is actually the method which does the deletion - call it `deleteCropCycle` or `deleteConfirmationHandler`
2. The method currently named `deleteCropCycle` is not actually doing the deletion work but just opening the modal. Can we name it `deleteClickHandler`. Should also rename the `editCropCycle` to `editClickHandler`.